pallets/nbv-storage/README.md

```diff
@@ -0,0 +1 @@
+License: Unlicense
```

Comment:
In the pallet README, it would be helpful to add example `polkadot-js-api` CLI commands that submit extrinsics and also read the pallet state. It is helpful for JS developers to be able to integrate the functionality with UI screens.

Reply:
Got it. I made all the suggested changes, including the documentation